.github/workflows/release.yml

```diff
@@ -19,6 +19,9 @@ jobs:
         with:
           python-version: '3.13'
           cache: true
+          cache-dependency-path: 'pylock.toml'
+      - name: Use pylock.toml
+        run: pdm config lock.format pylock
```

Comment:
Unfortunately, PDM doesn't recognize the new lock file without telling it to explicitly look for it.